src/cocol/node/ledger/action.cr

```diff
@@ -1,34 +1,45 @@
-module Ledger::Model
+module Ledger::Action
   alias TxnHash = String
 
-  abstract struct AbstractTransaction
+  struct Signature
     include JSON::Serializable
 
+    getter v : String
+    getter r : String
+    getter s : String
+
+    def initialize(@v, @r, @s)
```

Comment:
shall we implement this in secp256k1.cr?

https://github.com/q9f/secp256k1.cr/issues/35

Reply:
definitely! :)